policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -488,6 +488,15 @@ fn parse_accrual_config(
                         "min_penalty ({min_penalty:?}) cannot exceed max_penalty ({max_penalty:?})",
                     );
                 }
+                if max_penalty == time::Duration::from_millis(0) {
+                    bail!("max_penalty cannot be zero");
+                }
+                if jitter < 0.0 {
+                    bail!("jitter cannot be negative");
+                }
+                if jitter > 100.0 {
+                    bail!("jitter cannot be greater than 100");
+                }
```

Comment:
nit, take it or leave it: `anyhow` also has an [`ensure!` macro](https://docs.rs/anyhow/latest/anyhow/macro.ensure.html) which is essentially equivalent to `if !... { bail!(...) }`; so we could use that here?
```suggestion
               anyhow::ensure!(max_penalty > time::Duration::from_millis(0), "max_penalty cannot be zero");
               anyhow::ensure!(jitter >= 0.0, "jitter cannot be negative");
               anyhow::ensure!(jitter <= 100.0, "jitter cannot be greater than 100");
```

Reply:
ah that's neat :+1: